qpdf/qtest/qpdf/test-89.out

```diff
@@ -1,5 +1,5 @@
 WARNING: manual-qpdf-json.json, trailer at offset 1801: operation for array attempted on object of type dictionary: ignoring attempt to append item
 WARNING: manual-qpdf-json.json, obj:1 0 R at offset 1119: operation for array attempted on object of type dictionary: ignoring attempt to append item
 WARNING: manual-qpdf-json.json, obj:5 0 R at offset 1444: operation for dictionary attempted on object of type array: ignoring key replacement request
-WARNING: manual-qpdf-json.json, obj:5 0 R at offset 1456: operation for dictionary attempted on object of type name: ignoring key replacement request
+WARNING: manual-qpdf-json.json, obj:0 0 R at offset 1456: operation for dictionary attempted on object of type name: ignoring key replacement request
```

Comment:
How did we lose the object number here?

Reply:
See my comment above from just now. The object at at 1456 is direct.